packages/components/src/components/grid.tsx

```diff
@@ -17,8 +17,9 @@ type Props = {
     fetchGifs: (offset: number) => Promise<GifsResult>
     onGifsFetched?: (gifs: IGif[]) => void
     onGifsFetchError?: (e: Error) => void
+    noResultMessage?: string | HTMLElement
```

Comment:
@JeffersonFilho no luck with `string | JSX.Element` here?

Reply:
Weird... When i tried it typescript wasn't liking it, but i just tried again and seems to be fine now, i will replace with `JSX.Element`